src/west/_bootstrap/main.py

```diff
@@ -136,28 +121,28 @@ def init(argv):
 
 In more detail, does the following:
 
-  1. Clones the manifest repository to west/manifest, and the west repository
-     to west/west
+  1. Clones the manifest repository to a temporary folder, and the west
```

Comment:
Can you change the text to reflect the order this is being done in?
`Clones the west repository to .west/west and the manifest repository to a temporary folder`.

Reply:
This is the order.
We clone the manifest repository first, in order to know which revision / from where `west` must be cloned.

Reply:
Oops you are completely right, sorry.